tests/frameworks/test_huggingface.py

```diff
@@ -173,7 +173,7 @@ def test_huggingface_log_model(
     trainer.train()
 
     expected_call_count = {
-        "all": 2,
+        "all": 3,
```

Comment:
@dberenbaum when you have a minute - for some reason there are more calls. I suspect that removing catalyst made it run with the recent version of the HF transformers. 3 calls come from them, AFAIU we don't control it much>